ledger/eval.go

```diff
@@ -739,9 +742,10 @@ func (eval *BlockEvaluator) prepareEvalParams(txgroup []transactions.SignedTxnWi
 			PooledApplicationBudget: &pooledApplicationBudget,
 			FeeCredit:               &credit,
 			Specials:                &eval.specials,
+			FirstValidTimestamp:     uint64(ts),
 		}
 	}
-	return res
+	return res, nil
```

Comment:
I think you've removed the need to return error, so let's go back to the old signature.

Reply:
thanks, overlooked that,